### PR TITLE
Update flags for the Tezos baking app

### PR DESCRIPTION
**Pull request by spalmer25**
- Remove `LIBRARY` flag as the app is not called as a library
 - Set `BOLOS_SETTINGS` flag as the app needs lock screen settings
 - Set `GLOBAL_PIN` flag as the app needs pin validation

### CODE REVIEW

File: app-load-params-db.json [hunk 1]
```diff
@@ -1733,7 +1733,7 @@
     "path": ["44'/330'"]
   },
   "tezos_baking": {
-    "appFlags": {"nanos": "0x800", "nanos2": "0xa40", "nanox": "0xa40", "stax": "0xa40"},
+    "appFlags": {"nanos": "0x040", "nanos2": "0x040", "nanox": "0x040", "stax": "0x040"},
```
Comment (xchapron-ledger) [inline app-load-params-db.json:1736]: Their won't be BLE support on NanoX and Stax? If you want it, you need 0200 flag on them.

Comment (spalmer25) [reply app-load-params-db.json]: Oh yes, of course!
Thank you so much!

Comment (spalmer25) [reply app-load-params-db.json:1736]: We do not wish to support `BLE` for _NanoX_ and _Stax_, especially for the Baking app, as we really want the devices to remain plugged in.

**Review by ajinkyaraj-23 — APPROVED** [review]
LGTM

**Comment by ajinkyaraj-23** [comment]
Can we get this merged please @xchapron-ledger ?

**Review by tdejoigny-ledger — APPROVED** [review]

**Pull request merged**